src/lib/api.js

```diff
@@ -350,7 +348,7 @@ const addItemLinks = function (results, endpoint) {
 }
 
 const collectionsToCatalogLinks = function (results, endpoint) {
-  const stacVersion = process.env.STAC_VERSION
+  const stacVersion = process.env.STAC_VERSION || '1.0.0'
```

Comment:
if STAC_VERSION was unset, all the the STAC objects had no `type` field and were invalid.